solutions/fully-configurable/main.tf

```diff
@@ -43,3 +42,8 @@ module "kms" {
   keys                              = var.keys
   cbr_rules                         = var.key_protect_instance_cbr_rules
 }
+
+moved {
```

Comment:
This moved block is pointless - there is not upgrade path since we are changing the variation name and bumping the major version

Reply:
removed since we switched back